site/docs/master/upgrade-to-1.1.md

```diff
@@ -0,0 +1,36 @@
+# Upgrading to Velero 1.1
+
+## Prerequisites
+- [Velero v1.0][1] installed.
+
+Velero v1.1 only requires user intervention if Velero is running in a namespace other than `velero`.
+
+## Instructions
+
+### Adding VELERO_NAMESPACE environment variable to the deployment
+
+Previous versions of Velero's server detected the namespace in which it was running by inspecting the container's filesystem.
+With v1.1, this is no longer the case, and the server must be made aware of the namespace it is running in with the `VELERO_NAMEPACE` environment variable.
+
+`velero install` automatically writes this for new deployments, but existing installations will need to add the environment variable before upgrading.
+
+To do so, use `kubectl edit`.
+
+```bash
+kubectl edit deployment/velero -n <YOUR_NAMESPACE>
+```
+
+Add the following snippet:
+
+```yaml
+...
+        env:
```

Comment:
maybe just add a comment indicating this is under `.spec.template.spec.containers[0]`. Alternately, you might be able to come up with a `kubectl patch` command here, but that's not essential (may be tricky since it'd be appending to a list)

Reply:
Updated with a patch command.

Reply:
Awesome - worked for me as well.